mantidproject/mantid
# Get FindNumpy working with python3

FindNumpy.cmake doesn't work with python3 because the print syntax has changed.

```
CMake Error at buildconfig/CMake/FindNumpy.cmake:20 (if):
  if given arguments:

    "NOT" "File \"<string>\", line 1
      import numpy" " print numpy.get_include()
                              ^

  SyntaxError: invalid syntax" "MATCHES" "Traceback"

  Unknown arguments specified
Call Stack (most recent call first):
  Framework/PythonInterface/CMakeLists.txt:18 (find_package)


-- Configuring incomplete, errors occurred!
```

**To test:**
- Run cmake on a system with python3 and see if it finds numpy. You can run cmake defining which python to use like `cmake -DPYTHON_EXECUTABLE=/usr/bin/python3 .`
- Make sure it still works with python2, _i.e._ the builds pass

_Does not need to be in the release notes._

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
